src/main/java/org/jenkinsci/plugins/xunit/XUnitProcessor.java

```diff
@@ -47,13 +49,16 @@
 import org.jenkinsci.plugins.xunit.threshold.XUnitThreshold;
 import org.jenkinsci.plugins.xunit.types.CustomType;
 
+import javax.annotation.CheckForNull;
 import java.io.File;
 import java.io.IOException;
 import java.io.Serializable;
+import java.util.Arrays;
 
 /**
  * @author Gregory Boissinot
  */
+@SuppressFBWarnings(value="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE", justification = "generatedJunitDir.mkdirs() result is not needed")
```

Comment:
I would declare it on the method level at least. Generally this method checks whether the folder creation was successful, so I kinda question the justification